### PR TITLE
FIX: Sound mixer auto reset

### LINKED ISSUE #2747: [MicroWin] Sound precentage resets itself every time.
## Describe the bug

Let's say Google Chrome had it's sound to 100% and you set it to 30% (or any percentage) in Volume Mixer, if you close and re-open Google Chrome the sound goes back to 100% by itself no matter the application.

## Expected behavior

The sound should not reset itself at all.

## Screenshots

No need to I think

## Additional context

So the problem is, after you make your ISO with MicroWin and install the windows, if you set your volume mixer to any application/program after you close it or restart/shutdown your PC the to the specific application or even all of them the sound goes back to 100% every time and it becomes extremely annoying to deal with no matter the Windows Version (10/11).

### LINKED ISSUE #2776: Sound mixer settings keeps resetting
## Describe the bug
I used MicroWin to install a debloated version of Windows 11. With this version, my sound mixer does not save the changes when I adjust the volume (e.g., system sounds, Discord, etc.). I've seen others with the same issue using MicroWin. Applying the fix mentioned in issue #2747 helps save the volume adjustments after closing the sound window or restarting the PC. However, if I receive a notification from any application, the sound settings reset.
 

## To Reproduce
1. Debloat Windows 11 with MicroWin.
2. Install the debloated Windows 11.
3. Let Windows download and install all updates.
4. Restart your PC as many times as needed for the updates.
5. Adjust the system sound in the sound mixer.
6. Close the window.
7. Open the sound mixer window again (at this point, the settings should already be reset because the fix hasn't been applied yet).
8. Apply the fix mentioned (this should work after closing the sound window and restarting the PC).
9. Trigger any notification from Windows (the sound settings should reset again).
10. Open the sound mixer and check that all changes have been reset.

## Expected behavior
Sound adjustments should been saved.

## Screenshots
-

## Additional context
-

### PR DESCRIPTION
**Pull request by MyDrift-user**
## Type of Change
- [x] New feature

## Description

- fix issue nonrelated to microwin but windows itself [since at least 9 years](https://www.reddit.com/r/techsupport/comments/4h9vjt/solution_to_windows_10_volume_mixer_not_saving/) , duh.
  - add reg addition of key to make sound mixer work
    - [Here](https://www.reddit.com/r/ChrisTitusTech/comments/1e656ex/sound_mixer_not_saving_individual_app_sound) is the link to the reddit post about it from r/christitustech
- remove unused var definitions (from removed Microwin Options)

## Testing

- test worked for others, but you have to restart twice for it to save the application volume correctly like explained in reddit thread.

## Issue related to PR

- Resolves #2776
- Resolves #2747

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Review by CodingWonders — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Review by ChrisTitusTech — CHANGES_REQUESTED** [review]
We need to load the HKCU Hive using a reg load command

Right now only HKLM is being loaded and can be changed. 

Once this is done we should be able to do a command like reg add "HKCU/zSoftware..."

**Comment by CodingWonders** [comment]
It's already fixed, @ChrisTitusTech

File: functions/public/Invoke-WPFMicrowin.ps1 [hunk 1]
```diff
@@ -238,6 +234,9 @@ public class PowerManagement {
         # Write-Host Error code $LASTEXITCODE
         Write-Host "Done disabling Teams"
 
+        Write-Host "Fix Windows Volume Mixer Issue"
+        reg add "HKLM\zDEFAULT\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore" /f
```
Comment (CodingWonders) [inline functions/public/Invoke-WPFMicrowin.ps1:238]:
```suggestion
        reg add "HKLM\zNTUSER\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore" /f
```

See the reason why in the Discord DM

**Review by CodingWonders — CHANGES_REQUESTED** [review]

File: functions/public/Invoke-WPFMicrowin.ps1 (hunk 1 above)
Comment (MyDrift-user) [reply functions/public/Invoke-WPFMicrowin.ps1]: And the "software" does not have to be after zuser and microsoft?

Comment (CodingWonders) [reply functions/public/Invoke-WPFMicrowin.ps1]: > And the "software" does not have to be after zuser and microsoft?

No. `zNTUSER`, which is basically the user's `NTUSER.DAT` file, contains the majority of HKCU (the SOFTWARE key).

So you're already loading the SOFTWARE hive

**Comment by MyDrift-user** [comment]
This change is now included in the linked anniversary PR by @CodingWonders, so I will be closing this.

**Comment by jflores-p** [comment]
Hi, arriving a bit late, but i used the utility with the 25h2 iso and I'm getting the same behavior, with brave and discord notifications in my case (the main sound of discord does get saved, but the muted/deafe/etc sounds do reset after relaunch)
System sounds and Lost Ark audio are being saved so far.

I guess this could be because of the new ISO, but not so sure, not so savvy in this regard.

I have some entries in 'Computer\HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore' and see some updates in the corresponding folders for system sounds and Lost Ark.
I tried hard copying the settings of system sounds in the folder corresponding to brave but windows doesn't seem to read it. Not sure if it would work regardless, I just inferred it.

Any tips or recommendations? thanks :D

PS. also commented in #2747 just in case

**Pull request closed without merging**